### PR TITLE
#633: Update the documentation for version 3.5.0

### LINKED ISSUE #633: Update User Guide for version 3.5.0
I have updated the User Guide for 3.5.0, edits are marked in yellow in the google document here: https://docs.google.com/document/d/1XzBC65fvuNqCNhctJgDicPKqNWh0DsPucVByly9kiU0/edit#heading=h.ia0ex66ykr5q

### PR DESCRIPTION
**Pull request by matthew-morris-cessda**
This closes cessda/cessda.cdc.versions#633

### CODE REVIEW

File: detailed-study-view.md [hunk 1]
```diff
@@ -24,8 +24,7 @@ The infotips (highlighted here with a red circle in the image below)
 contain the names and links to these vocabularies.
 
 Below the abstract section, you will find topics and keywords.
-If you see a topic or keyword relevant to you (e.g. “Drinking behaviour”), you can click on it.
-This will initiate a search using this topic or keyword in the topic or keyword filters.
+If you see a topic relevant to you (e.g. “General health and well-being”), you can click on it. This will initiate a search using this topic in the topic filter. If you see a keyword that is relevant to you (e.g. “Health status”) you can click the keyword to initiate a search using this keyword in the keyword filter. Or you can click on “ELSST” to view the keyword in the European Language Social Science Thesaurus to read more information about the keyword and find related keywords.
 
 ![Detailed study view](images/detailed-study-view.png "Detailed study view")
```
Comment (john-shepherdson) [inline detailed-study-view.md:33]: Need to replace existing image file with one showing links to ELSST

Comment (john-shepherdson) [reply detailed-study-view.md:33]: Has the image file been updated?

Comment (matthew-morris-cessda) [reply detailed-study-view.md:33]: The image has been updated

**Review by john-shepherdson — APPROVED** [review]

**Comment by john-shepherdson** [comment]
@MortenSikt Updates can be seen at https://datacatalogue-dev.cessda.eu/about and https://datacatalogue-dev.cessda.eu/documentation/

**Comment by john-shepherdson** [comment]
@matthew-morris-cessda About page has wrong year in date. Change from 2023 to 2024.

**Comment by matthew-morris-cessda** [comment]
Just noticed this, a fix is being deployed now

**Comment by matthew-morris-cessda** [comment]
Fixed on staging

**Pull request merged**